contents/blog/posthog-vs-fathom.mdx

```diff
@@ -0,0 +1,168 @@
+---
+title: "In-depth: PostHog vs Fathom"
+date: 2024-11-11
+author:
+ - bijan-boustani
+rootpage: /blog
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/posthog_vs_fathom_analytics_e327b58767.jpg
+featuredImageType: full
+category: General
+tags:
+  - Comparisons
+---
+
+import { ComparisonTable } from 'components/ComparisonTable'
+import { ComparisonHeader } from 'components/ComparisonTable/header'
+import { ComparisonRow } from 'components/ComparisonTable/row'
+
+PostHog and Fathom Analytics are both alternatives to Google Analytics. In fact, both explicitly call this out on their websites with marketing copy like "Switch to a better Google Analytics alternative."
+
+Both products have web analytics dashboards with an emphasis on simplicity and privacy. They're ideal choices for customers that want to analyze and understand web traffic but find Google Analytics to be too complicated.
+
+## How is PostHog different?
+
+### 1. PostHog offers web analytics, and a lot more
+
+[Web analytics](/web-analytics) is a core feature of PostHog, but we offer many more tools to help you analyze, test, and observe your site and your product.
+
+These include [product analytics](/product-analytics), [session replay](/session-replay), [surveys](/surveys), [experiments](/experiments), [feature flags](/feature-flags), a [data warehouse](/data-warehouse), and more.
+
+### 2. PostHog has a generous free tier
+
+PostHog's ["ridiculously cheap" tier](/pricing) includes all of our features for free with an allowance of 1M events per month. For Fathom, pricing starts at $15 per month for 100K data points, increases to $60 per month for 1M data points, and more from there.
+
+Beyond PostHog's free tier, we try to be as cheap as possible and [transparent](/docs/billing/estimating-usage-costs). Want to know how much we'll charge? See our [pricing calculator](/pricing#calculator). PostHog also does not require a credit card to sign up and try the free tier.
+
+## Web analytics
+
+<ProductScreenshot
+  imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/web_analytics_dashboard_light_bc76a94566.png"
+  imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/web_analytics_dashboard_dark_42ac39a5f7.png"
+  classes="rounded"
+  alt="PostHog web analytics dashboard"
+/>
+
+![Fathom Analytics dashboard](https://res.cloudinary.com/dmukukwp6/image/upload/fathom_analytics_dashboard_195049b289.png)
```

Comment:
This is a lot of screenshots to have all at once, maybe move them below the table with some context on why you are including them? For example:

"In terms of design, both PostHog and Fathom's web analytics dashboards look similar..."

(PostHog screenshot)

"The big difference is that PostHog includes sessions where Fathom only includes visitors"

(Fathom screenshot)